packages/api-request-builder/test/create-service.spec.js

```diff
@@ -88,61 +89,283 @@ describe('createService', () => {
     )
   })
 
+  describe('parse', () => {
+    let service
+    beforeEach(() => {
+      const options = {
+        type: 'foo',
+        endpoint: '/foo',
+        features: [
+          defaultFeatures.query,
+          defaultFeatures.queryOne,
+          defaultFeatures.queryExpand,
+          defaultFeatures.projection,
+        ],
+      }
+      service = createService(options, projectKey)
+    })
```

Comment:
I extracted the service creation into `beforeEach` so that we don't have to repeat it.